storage/bgmetadata_backend.go

```diff
@@ -0,0 +1,27 @@
+package storage
+
+type BgMetadataStorageConnector interface {
+	UpdateMetricMetadata(metric Metric) error
+	InsertDirectory(dir MetricDirectory) error
+	SelectDirectory(dir string) (string, error)
+}
+
+type BgMetadataNoOpStorageConnector struct {
+}
+
+func NewBgMetadataNoOpStorageConnector() *BgMetadataNoOpStorageConnector {
```

Comment:
I don't think this method is really needed as we just initialized empty struct

Reply:
fair enough